### PR TITLE
Add neo4j example to DC/OS examples

### PR DESCRIPTION
**Pull request by unterstein**
Examples for https://github.com/mesosphere/universe/pull/863

### CODE REVIEW

**Comment by mhausenblas** [comment]
Awesome, thanks @unterstein!

**Comment by ryadav88** [comment]
@mhausenblas this is ready to review

File: 1.8/neo4j/README.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+$ dcos package uninstall neo4j; dcos package uninstall neo4j-replica; dcos package uninstall neo4j-proxy
+```
+
+### Configuration options
+
+There are a number of configuration options, which can be specified in the following
+way:
+
+```bash
+$ dcos package install --config=<JSON_FILE> arangodb3
```
Comment (mhausenblas) [inline 1.8/neo4j/README.md]: yeah I guess you want to change this? ;)

**Comment by mhausenblas** [comment]
Thanks @ryadav88 and looks great to me @unterstein just one thing to change, left a comment in-line.

**Comment by mhausenblas** [comment]
Thank you!

**Pull request merged**